src/sdk/__contracts/addresses.ts

```diff
@@ -4,6 +4,10 @@ export const addresses = {
   Nexus: "0x21f4C007C9f091B93B7C1C6911E13ACcd3DAd403",
   K1Validator: "0x6854688d3D9A87a33Addd5f4deB5cea1B97fa5b7",
   K1ValidatorFactory: "0x976869CF9c5Dd5046b41963EF1bBcE62b5366869",
-  UniActionPolicy: "0x28120dC008C36d95DE5fa0603526f219c1Ba80f6"
+  UniActionPolicy: "0x28120dC008C36d95DE5fa0603526f219c1Ba80f6",
+  TimeframePolicy: "0x0B7BB9bD65858593D97f12001FaDa94828307805",
+  // Note: deployed smart sess on 17 apr with useRegistry:false
+  SmartSession: "0x3834aD7f5f73fAd19C089a924F18e6F3417d1ac2",
```

Comment:
I was hoping this file would be the addresses that we expect to use in prod, and should be consistent from chain to chain. Do these new contract addresses fit that description?

Reply:
all the Note words comments are to be removed.